src/components/Modals/SelectLoomBall.tsx

```diff
@@ -0,0 +1,107 @@
+import { getItemsService } from '@src/services/items.services';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Modal from 'react-native-modal';
+import { CustomButton } from '@src/components/CustomButton';
+import { TLoomball } from '@src/types/types';
+import { SelectLoomballGrid } from '../ItemsGrid/SelectLoomballGrid';
+
+export interface iPropsSelectLoomBallModal {
+  isVisible: boolean;
+  toggleVisibilityCallback: () => void;
+  submitCallback: (loomBall: TLoomball) => void;
+}
+
+export const SelectLoomBallModal = ({
+  isVisible,
+  toggleVisibilityCallback,
+  submitCallback
+}: iPropsSelectLoomBallModal) => {
+  const [loomBall, setLoomBall] = useState<TLoomball[]>([]);
+  const [selectLoomball, setSelectLoomball] = useState<TLoomball>();
+
+  //Get loomBall from user
+  const getUserLoomBalls = async () => {
```

Comment:
This is sending a request to the backend **every time the user changes the Loomball**. We don´t need that and and that can turn into a huge performance issue. 

I'll fix this, but be careful with the API calls and the `useEffect` hook.